enhancements/baremetal/enable-baremetal-on-other-platforms.md

```diff
@@ -0,0 +1,197 @@
+---
+title: enable-baremetal-on-other-platforms
+authors:
+  - "@asalkeld"
+  - "@sadasu"
+reviewers:
+  - "@hardy"
+  - "@romfreiman"
+  - "@dhellman"
+approvers:
+  - "@hardy"
+creation-date: 2021-08-20
+last-updated: 2021-08-20
+status: implementable
+see-also:
+  - "/enhancements/baremetal-provisiong-config.md"
```

Comment:
should be `/enhancements/baremetal/baremetal-provisiong-config.md`

Reply:
Or maybe this can be a relative path?  Either way I think the current link is broken :)